include/api.php

```diff
@@ -3089,8 +3090,8 @@ function api_fr_photos_list($type) {
 	}
 
 	function api_fr_photo_detail($type) {
-		if (api_user()===false) throw new ForbiddenException();
-		if(!x($_REQUEST,'photo_id')) throw new BadRequestException("No photo id.");
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.